yarn.lock

```diff
@@ -0,0 +1,3985 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
```

Comment:
I don't use `yarn`, so this shouldn't be checked in. And `package-lock.json` should be updated. I've already done that in https://github.com/idkjs/bs-revamp/pull/1 though

Reply:
Thank you for this detailed guidance, sir. I had checked in the yarn.lock because I could not get this to build with npm for some reason. Initially, it was the old bs-platform so i bailed out and used yarn. I tried to rebuild with npm afterwards but still did not work so...yarn. Thanks for fixing it. 

Thanks for pointing this out and the other comments you made. 

Your comment about formatting above is one I have heard before. How do you keep your editor from changing the formatting? Usually, I am in vscode. Should I take out any formatting configuration or is the solution something else? On "serious" commits, I find myself going through the output and fixing the unnecessary formatting changes where there isnt a `.prettierrc` or linter in the project.

Reply:
You should be able to just set `"editor.formatOnSave": false` in your `settings.json`. IMO format on save ought to be disabled globally by default, and only enabled per-project. That'll avoid any accidental formatting.